services/movie_session.py

```diff
@@ -42,3 +43,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
You should use the `get_movie_session_by_id` method to get the required `MovieSession` object.